src/modules/70-pipeline/pages/webhooks/WebhookEvents/WebhookEvents.tsx

```diff
@@ -177,6 +208,19 @@ export default function WebhookEvents(): JSX.Element {
               })
             }}
           />
+          <MultiSelectDropDown
+            width={120}
+            buttonTestId="status-select"
+            value={actualValue}
+            onChange={option => {
+              updateQueryParams({
+                eventStatus: (option.flatMap(item => item.value) as WebhookEventStatus[]) || null
+              })
+            }}
+            items={getStatusList(getString)}
+            usePortal={true}
```

Comment:
No need for the `={true}`
```suggestion
            usePortal
```